test/e2e/fixture-builder.js

```diff
@@ -169,7 +169,7 @@ class FixtureBuilder {
:"1bMxVc4PgogxGePtaCqfWAm3/ASfbOwnw6kGskZFqNoc/vSECsDK7eeWHZpybG2lG29KBQzMfDyy0Dt1kNcsU8DyP6LdJtSzOwm4TLu9QoSUSM8X365ae2zShi2MJfKHvqCRwhx4Dcx90Gz+Ddx2pmLjmMU2RWFC50cnKo+p1Ml3f7XaC+BqLRrpGipKolY7GJDCA/NFvW6JAYWsBOrQKClswb2xAh5AcZRIVT4m31XVEfJNT8z1Z+6J8xZu1vSJd6UEl6PWF53gu4aBAYvSpshXHqMG76WbAGvwS+7l53pC1VGN5TNY3bu+f6H03HXBMXkhfscVf1LljNSiKuHPsZoCNCegtiqEjQwqpGLDKHyujq0NsL30lBk7B1uclWWdVY3Wh2eTFO7ZyS+hjIeSLcTiY4bb1fhHzwbu/YrkkStCH9Z3OGWqyDqWU96He4mMqcsz2dOM3yOXBQmKQSVgDqGCiC3KIrs7y2hotkPjgy+FESVmZfQKAVRtEjwUL7wO68E+xBAizIPQJlvOsBFAN/Cdk487h1xDg3IICQnNe2n1e9RmRXw9VYs6UygXVZ2xBWT5H8BSLzXXVzWC/OoQeEOC4FmMwNQFTo3sJUEFtR3weV+TxR4yXpjSY11nRbw1IBMlMZUjlb4+36APGGXpQbyeXGKmyjdB6v51GbMfhAfy+r/O8ScL2ME8RCB00o6Kyoet5UDC9uNSDlRIjLnINWjQtapCPr5UYqaT7uQ2wNdI4rRhz7HWc4q+arDPJdnktt1pPYsbQPK4/98H8luZkD7nXhwerSP9QFrbH/Qjj5IjBiopY0znNx4y82WKuuehSmG8fEnA+YlqaC+paOG76yx3Wj8jkKM26ORnI06co54821ajutv7i+VRV/4RtD5bSjurEkisyKlywp9JDWx3Y2JeA/xovWFA4Rc4cvTAiiO0m6ywhxvP/K8Tv2QwO2BA6kiJEX5lHRFIt1nOlQvZ5r8tEMNpgu4OgFE2paDzIzoNC3aEpUASxt3PsfYdn9/iHt7vN+JZGwWRANJDVKnp1ZJcWu2lyQc5oBLN32el5wlMpHNYa8c7DJ+6h8FEM2b+ZvpNaaL1usQZo7i6VwPU8yDWbx+ieMAB0Vis5S+HwUcRKSIuWnGMZTbtyF53d9ch0f7nLB8W0i2Qji1uPMcDoJnn33TJsUpFN1QISGy3vjHBjnbSVXEkLIYZIJ1hSNgzfzJ+6xxkOISDVgDqwJqMpcqe3YahfVt762eFp/ke7kGKftsrr86qRczSZR4kHWryO6022rouwWHHVjqOsBahqKisMARomFUS2BtEG8vG1VIlT9gAzgOvi2qcZfd/r9s3TayJuRst3k13NUm4s1nqoN7JG3v3cyNyGSVRpTr68wqa0QW+gZ+NY77inj6Drvs3THotK4aCcwMXIEtmEWquX5F4AcJgsRIuCl4ZYwAAwR/behTASFXe4WkM8eEdx+xBoq5iJQid4hP6WqV+IPkRkX2/nVqinqFExYgHJ7RemK5UUIkKDm2bTzMuItoUYiH+HLfFNXhecVAIYUBqcPNjRSm2qVq35qLTVlLuAx/K3DP/e/LjPcTdSR9rnaVumLtna8gDCrOE0AoJjyIMjZgA690v3vzpvN+XIOrCNMyElZS8sc6nZqj/RfMx1grZgGTT7m7O+8rVTeKibIZ6FFD4f0UEMtN37I0ioR7Cw9pRhAA+T9Viq6baYglHyg/k1y2brNMEWB0tGzJxUBlJOTaDtHwtgPnMlPqvcPMh66q/D+i2Moh2oFoXlE28vLMUDhEPhfT+7Md+d1I8P48gIC3Te5HbbdgjqfjHH/SIdTTIYVbpBnYDtiOhktyopXmfqIPqdiv10Wv9UuyqJrfo7Vejn9WXMo6p+dhBqO6WQKnalC+gp3wJA0X5YPVnbvoBzdT6Suclk+dPUzhYIHRIMebamC0Ps8d+J+qZhxJgGmL5m8XaSXPmpBBQpZZR5cu4GoVaf9kUpCStn3mvvdDLPWVamlmGpAx+uQ6H/dvOFV2u2sjUVRTuWhnrQXaFy0rdokgQCUHbCSoq+0cu/oE7k87U+YLRPRjJYrdPXWAzLh/YIZUPZm+nA3bWXp+MC0YUxb1DdIfKIGnfSbTdk/uPJBGHlUp1cUL4Szy7em1Ut1KGivTuLhgwSvPtSfDglaHZf7DRGYGJ4dZs9tEjvrNfOvVkipl+fxaRuPeQ7i+VDa2J2eKpY2zIYPQdTKIfJ7w1ZVNDlwcF+dfyxEibh5fQJNt7uWFQfTaSwzSFyeKCv1gr3wJHFTtVqYxE6yDOG0b5YbbSrtlULc2OoyJXwCCWTGRCZXSlQdb1iozavcrINOd6powTgI0dB1+vURCn3HqfHctRNm7V0sCu7Yp1dpqIgyQSjwSoO+Y2yWT2PqcCiBwN/JBlEdtxT5LVRnIrcBvQ+FnxXYuHnn+s+wbfSeqZh03njXIe9xRLcUO/DvJSiKSFVe+jyehRMC9xrL1Pc3Py1WmMLZ/iYv0p6hcwpV5sA5guXmmJkkpd5rt6r+6wW+JT/X/uscDXvc8QiyUFu83KftM8HE0csbaNtt1JX5UAFrONwuUT17fkpmP+5iX3818F2k7GXQucoJ1aAHJDM2cuOQpbPEq7NDldG+9UM3uuuYfrSxbnpkGN8QYgiDkg/xSNuXKtL0MO70rKSRdf0k6xMQJgDJS1WvKGu8107MIgRHbxZv7rAnPZB7EjlxhVgFDN/TscCupfSNmrMvEDNmunIL6sPkwENIGpifzbeR7HIzdBfdFwC/qqopCibOuymGm3edWmtYsEilI8JhHrI8+2nEs3jQ0gh5DxqeqFf34dp7nNpk4=","iv":"06oxzy+APV60DhKCINN5Cw==","keyMetadata":{"algorithm":"PBKDF2","params":{"iterations":600000}},"salt":"qmOjcRo7WB/mEWTtrYQ1YomdCvUz48Scf7vYBge0V4M="}',
```

Comment:
There are two different `SnapKeyring` in this vault, it should never be possible. Not sure if it is causing any issue, but probably best to remove the second one so we avoid subtle issues in the future

```
[
  ...,
  {
    "type": "Snap Keyring",
    "data": {
      ...
    },
    "metadata": {
      "id": "01K90ME8BAWA3BCEHPDHB0SGG7",
      "name": ""
    }
  },
  {
    "type": "Snap Keyring",
    "data": {
      "accounts": {}
    },
    "metadata": {
      "id": "01K90ME8BB6N91MZ5ARVPSEM3J",
      "name": ""
    }
  },
  {
    "type": "HD Key Tree",
    "data": {
      ...
    },
    "metadata": {
      "id": "01K90MKV5XAH4Q3F75MFHKBX75",
      "name": ""
    }
  }
]